browser/src/NeovimInstance.ts

```diff
@@ -126,6 +126,14 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                             this.emit("event", eventName, eventContext)
                         } else if (pluginMethod === "window_display_update") {
                             this.emit("window-display-update", args[0][0], args[0][1])
+                        } else if (pluginMethod === "api_info") {
+                            const apiVersion = args[0][0]
+                            if (apiVersion.minor < 1) {
```

Comment:
@justinmk sorry, I didn't follow the whole API level versioning discussion closely : the version numbers are not semantic? Then my comments are academic, if course: true as to pure code logic but irrelevant to the issue at hand ;).